README.md

```diff
@@ -257,18 +257,27 @@ Choo fires messages when certain events happen:
 - __`.on('pushState')`__: when the history API is triggered
 
 ### `app.route(routeName, handler)`
-Register a route on the router
+Register a route on the router. Uses [nanorouter][nanorouter] under the hood.
+Params can be registered by prepending the routename with `:routename`, e.g.
+`/foo/:bar/:baz`. The value of the param will be saved on `state.params` (e.g.
+`state.params.bar`). Wildcard routes can be registered with `*`, e.g. `/foo/*`.
+The value of the wildcard will be saved under `state.params.wildcard`.
+
+Using hashes to delimit routes is supported out of the box (e.g. `/foo#bar`).
+When a hash is found we also check if there's an available anchor on the same
+page, and will scroll the screen to the position. Using both hashes in URLs and
+anchor links on the page is generally not recommended.
```

Comment:
may or may not be good to mention working with query parameters too? idk if folks normally use those (i do, so mentioning it)